Resources/Prototypes/_Harmony/Entities/Objects/Misc/tiles.yml

```diff
@@ -0,0 +1,48 @@
+# Xeno playtest
+- type: entity
+  name: dark techmaint floor
+  parent: FloorTileItemBase
+  id: FloorTileItemTechmaintDark
+  components:
+  - type: Sprite
+    state: techfloordark
+  - type: Item
+    heldPrefix: dark
+  - type: FloorTile
+    outputs:
+      - Plating
+      - FloorTechMaintDark
+  - type: Stack
+    stackType: FloorTileTechmaintDark
+
+- type: entity
+  name: reinforced tile
+  parent: FloorTileItemBase
+  id: FloorTileItemReinforced
+  components:
+  - type: Sprite
+    state: reinforced
+  - type: Item
+    heldPrefix: reinforced
+  - type: FloorTile
+    outputs:
+      - Plating
+      - FloorReinforced
+  - type: Stack
+    stackType: FloorTileReinforced
```

Comment:
Linter fail.

My best guess is `FloorTileReinforced` is missing from `Resources/Prototypes/_Harmony/Stacks/floor_tile_stacks.yml` or other one was supposed to be used.

Reply:
Turns out FloorTileReinforced was removed, likely due to there being no way to get this tile (since you just use a rod on an empty plating tile). So it 's an incorrect addition.